docs/style.css

```diff
@@ -1,3 +1,28 @@
+/* Colors */
+
+:root {
+    --color-topic-gebruikers: #d32f46;
+    --color-topic-principes: #a73276;
+    --color-topic-broncode: #684084;
+    --color-topic-technologie: #31406e;
+    --color-topic-proces-en-team: #1f3546;
+    --color-topic-ambacht: #316e6d;
+    --color-topic-overigen: #6eb542;
+
+    --color-black: #12121c;
+    --color-white: rgba(255, 255, 255, 0.95);
+    --color-orange: #f67905;
+
+    --color-background: var(--color-black);
+    --color-text-light: var(--color-white);
+    --color-text-dark: var(--color-black);
+    --color-block: #21212c;
+
+    --box-shadow: 5px 5px 20px rgba(0, 0, 0, 0.2);
+}
+
+/* Fonts */
+
```

Comment:
Nitpick: dat is wat mij betreft de extra bytes en schermruimte niet waard, uit de backets/opmaak/code zelf is dit al wel duidelijk?

```suggestion
```